website/docs/deploy/configuring-unleash.md

```diff
@@ -2,9 +2,8 @@
 id: configuring_unleash
 title: Configuring Unleash
 ---
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
 
+import Tabs from '@theme/Tabs'; import TabItem from '@theme/TabItem';
```

Comment:
While this doesn't break anything, I'd prefer it if the imports stayed on separate lines. Maybe that's a prettier setting?